src/util/github-utils.ts

```diff
@@ -14,6 +14,8 @@ export const isUserAdminOfOrganization = async (githubClient: GitHubUserClient,
 		return role === "admin";
 	} catch (err) {
 		logger.warn({ err, orgName, username }, `could not determine admin status of user in org`);
+		err.requiresSsoLogin = err?.cause?.response?.headers["x-github-sso"];
```

Comment:
Shouldn't error already be an instance of GithubClientSSOLoginError ? Why do we need this flag?

I think in github client interceptors we should check for:
 - status (=401?) 
 - the header

If these match, we rethrow the error as GithubClientSSOLoginError and then just use instanceof everywhere. Wdyt?

Reply:
Updated as suggested.